testing/bdd/features/timing/delay.feature

```diff
@@ -68,13 +68,16 @@ Feature: Delay of a document sequence
     | 00:00:05   |              |           |          |          |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
```

Comment:
In general for all of these the behaviour that we need to test is that the correct _specified_ times end up in the output document. I don't mind keeping these calculations of _computed_ times to help with debugging etc but we need the BDD to test the output documents' specified times as well.